components_new/asp/database/EventWriter.py

```
@@ -0,0 +1,92 @@
+    def checkIfFieldExists(self, table, column, value):
+        '''
+            Checks if the database has the given field.
+        '''
+        query = f"""
+            SELECT 1 FROM {table}
+            WHERE {column} = %s
+            LIMIT 1
+        """
+        self.cursor.execute(query, (value, ))
+
+        return True if self.cursor.fetchone() else False
```

Ruff (0.8.2)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)